api_reference.md

```diff
@@ -1083,40 +1079,6 @@ getNodeInfo()
     // ...
   })
 ```
-<a name="module_core.createGetTips"></a>
-
-### *core*.createGetTips(provider)
-
-| Param | Type | Description |
-| --- | --- | --- |
-| provider | <code>Provider</code> | Network provider |
-
-**Returns**: <code>function</code> - [`getTips`](#module_core.getTips)  
-<a name="module_core.getTips"></a>
-
-### *core*.getTips([callback])
-**Fulfil**: <code>Hash[]</code> List of tip hashes  
-**Reject**: <code>Error</code>
-- Fetch error  
-
-| Param | Type | Description |
-| --- | --- | --- |
-| [callback] | <code>Callback</code> | Optional callback |
-
-Returns a list of tips (transactions not referenced by other transactions),
-as seen by the connected node.
-
-**Example**  
-```js
-getTips()
-  .then(tips => {
-    // ...
-  })
-  .catch(err => {
-    // ...
-  })
-```
-<a name="module_core.createGetTransactionObjects"></a>
```

Comment:
This link seems to be broken now 🤔 

![image](https://user-images.githubusercontent.com/19519564/84583260-ac0dc400-adc4-11ea-9af4-f6b51926d94b.png)

Reply:
@rajivshah3 Good catch. Should be fixed with https://github.com/iotaledger/iota.js/pull/483/commits/ef28f8b5d919329df49180803a49509606d483a9